src/layerstack.stanza

```diff
@@ -536,3 +547,29 @@ public defn create-pcb-stackup (stack:LayerStack) :
       make-layer-statement(l)
 
   custom-stackup
+
+doc:\<DOC>
+  Construct a LayerStack with a tuple of outer layers. 
+
+  Example:
+  ```
+    val jlcpcb-jlc2313 = make-layer-stack("JLCPCB 4-layer 1.6mm", outer-layers) where :
+      val copper-35um = Copper(0.035)
+      val prepreg-2313 = FR4(0.1, FR4-Material-2313)
+      val copper-17_5um = Copper(0.0175)
+      val core = FR4(1.265, FR4-Material-Core)
+      val outer-layers = [
+        [copper-35um prepreg-2313] 
+        [copper-17_5um core]
+      ]
```

Comment:
Let's please add to the documentation here a description of what the stackup is expected to be after this function runs.

Reply:
Added it:
```
doc:\<DOC>
  Construct a LayerStack with a tuple of outer layers. 

  Example:
  ```
    val copper-35um = Copper(0.035)
    val copper-17_5um = Copper(0.0175)
    val jlcpcb-jlc2313 = make-layer-stack("JLCPCB 4-layer 1.6mm", outer-layers,
      soldermask = soldermask) where :
      val soldermask = Soldermask(0.019, SoldermaskMaterial)
      val prepreg-2313 = FR4(0.1, FR4-Material-2313)
      val core-2313 = FR4(1.265, FR4-Material-Core)
      val outer-layers = [
        [copper-35um prepreg-2313] 
        [copper-17_5um core-2313]
      ]
  ```
  The LayerStack created has these layers:
     [ soldermask
       copper-35um prepreg-2313
       copper-17_5um core-2313
       copper-17_5um core-2313
       copper-35um prepreg-2313
       soldermask
     ]
  @param name The name of the LayerStack
  @param top-layers The pairs of mechanical and dielectric layers
  @param description The description of the LayerStack
<DOC>
```

Reply:
Edited the comment above with the corrected layers.